javascript/node/selenium-webdriver/lib/command.js

```diff
@@ -229,6 +229,21 @@ class Executor {
    *     result.
    */
   execute(command) {}
+
+  /**
+   * Defines a new command for use with this executor. When a command is sent,
+   * the {@code path} will be preprocessed using the command's parameters; any
+   * path segments prefixed with ":" will be replaced by the parameter of the
+   * same name. For example, given "/person/:name" and the parameters
+   * "{name: 'Bob'}", the final command path will be "/person/Bob".
+   *
+   * @param {string} name The command name.
+   * @param {string} method The HTTP method to use when sending this command.
+   * @param {string} path The path to send the command to, relative to
+   *     the WebDriver server's command root and of the form
+   *     "/path/:variable/segment".
+   */
+  defineCommand(name, method, path) {}
```

Comment:
Alright.  Would you be interested in a CL which exposed `defineCommand()` on the `Builder`?